src/gnome-mastermind.c

```diff
@@ -1584,13 +1566,8 @@ static gboolean tray_mid_click(){
 
 //	 gm_debug ("i:%d ypos:%d c: %d l:%d\n", i, ypos, c, l);
 
-      if (timeout_id > 0) {
-	g_source_remove (timeout_id);
-      }
-      timeout_id = g_timeout_add (200, 
-				  (GSourceFunc) clean_tray, 
-				  drawing_area);
-
```

Comment:
it needs some more logic as timeout_id can be > 0 even if no timer is present, there probably was a bug here and Yavor just removed the code instead of trying to fix it. I'll see if there's an easy workaround.